### PR TITLE
Update table dependencies

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**
Just some maintenance work -- lest we fall behind and it becomes harder to maintain it up to date.

### CODE REVIEW

File: .config/webpack/base.js [hunk 1]
```diff
@@ -46,6 +46,14 @@ module.exports = (options = {}) => {
...
                     loader: 'raw-loader'
                 },
+                {
+                    test: /\.ts(x?)$/,
+                    include: /node_modules\/(highlight[.]js\/)/,
+                    use: [
+                        { loader: 'babel-loader', options: babel },
+                        { loader: 'ts-loader', options: ts },
+                    ]
+                },
```
Comment (Marc-Andre-Rivet) [inline .config/webpack/base.js:56]: highlight.js is now released in es6 - we need to transpile further

File: .config/webpack/base.js [hunk 2]
```diff
@@ -55,6 +63,13 @@ module.exports = (options = {}) => {
                         { loader: 'webpack-preprocessor', options: JSON.stringify(preprocessor) }
                     ]
                 },
+                {
+                    test: /\.js$/,
+                    include: /node_modules\/(highlight[.]js\/)/,
+                    use: [
+                        { loader: 'babel-loader', options: babel }
+                    ]
+                },
```
Comment (Marc-Andre-Rivet) [inline .config/webpack/base.js:72]: highlight.js is now released in es6 - we need to transpile further

File: .storybook/main.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    webpackFinal: async (config, { configType }) => {
+        // jerry rig everything
+        config.resolve.alias.core = path.resolve(__dirname, './../src/core'),
+        config.resolve.alias['dash-table'] = path.resolve(__dirname, './../src/dash-table')
+
+        config.module = baseConfig.module;
+
+        return config;
+    }
+};
```
Comment (Marc-Andre-Rivet) [inline .storybook/main.js:25]: Storybook 6 brings totally new configuration files and opinions. Making it work with our own needs..

File: tests/cypress/tests/standalone/markdown_test.ts [hunk 4]
```diff
@@ -140,7 +140,7 @@ describe('markdown cells', () => {
         it('loads highlight.js and does not attach hljs to window', () => {
             cy.visit(`http://localhost:8080?mode=${AppMode.Markdown}`);
             // wait for highlight.js to highlight code
-            DashTable.getCellById(0, 'markdown-code-blocks').within(() => cy.get('code.language-python span.hljs-title'));
+            DashTable.getCellById(0, 'markdown-code-blocks').within(() => cy.get('code.language-python'));
```
Comment (Marc-Andre-Rivet) [inline tests/cypress/tests/standalone/markdown_test.ts:143]: highlight.js 10 does not wrap the code in a span anymore, updating the test

File: .storybook/main.js (hunk 3 above)
Comment (rpkyle) [reply .storybook/main.js:25]: @Marc-Andre-Rivet For any package using Storybook >=6, I imagine we now have to add

```javascript
config.resolve.alias['dash-table'] = path.resolve(__dirname, './../src/dash-table')
```

but specific to the path/package name?

Comment (Marc-Andre-Rivet) [reply .storybook/main.js:25]: Yes. But the table is the only project using storybook.

**Pull request merged**